static/styles/night_mode.scss

```diff
@@ -501,6 +501,12 @@ on a dark background, and don't change the dark labels dark either. */
         }
     }
 
+    .selected_message .message_controls {
+        @media screen and (max-width: 350px) {
```

Comment:
I don't see any other media queries in the night_mode file.

I also don't see `.night-mode` being used alongside `@media`, in any .scss file.

@showell, I wonder where might be the best place to put day and night styles that only apply on small screens; I'm not sure this is the best place.

Reply:
Yeah, I'd prefer to just reference the night mode CSS class in the existing media CSS if we have to.